README.md

```diff
@@ -87,7 +89,206 @@ worker: `celery -A AIPscan.Aggregator.tasks worker --loglevel=info`
 
 ![screencap4](screencaps/aipscan_scatterplot_report.png)
 
+##Production intallation
+
+AIPscan is a Python [Flask][fla-1] application. The Flask Werkzeug development server is not recommended for use in production. Instead, a WSGI and HTTP proxy server should be used to serve the AIPscan application. We recommend [Gunicorn][gun-1] and [Nginx][ngx-1] respectively. The following instructions are for production deployment to an Ubuntu/Debian server. Other operating systems and servers have not been tested.
```

Comment:
Oh yep, yeah,, if these - just the narrative/instructional lines can all be converted to 80 chars max lines that would be great.